testing/configitemtest.cc

```diff
@@ -0,0 +1,285 @@
+#include <gtest/gtest.h>
+#include <gmock/gmock.h>
+
+#include "game/configitem.hh"
```

Comment:
this should appear first, as this would also test the header is self contained (does not need extra headers)

Reply:
I changed include and sequence.